torchvision/prototype/features/_feature.py

```diff
@@ -7,7 +7,52 @@
 F = TypeVar("F", bound="_Feature")
 
 
-class _Feature(torch.Tensor):
+class _TransformsMixin:
```

Comment:
Question: Interesting. You don't plan to include primitive transforms related to colour? Is this because only images support them?

Reply:
I haven't yet worked on that. But yes, color-related method will be there as well

Reply:
Yeap sorry, I read more into the absence of the methods than I should. 😅